ogr/services/gitlab.py

```diff
@@ -41,13 +52,16 @@ class GitlabService(GitService):
     def __init__(self, token=None, url=None, full_repo_name=None):
         super().__init__(token=token)
         url = url or "https://gitlab.com"
-        self.g = gitlab.Gitlab(url=url, private_token=token)
+        self.g = gitlab.Gitlab(url=url, private_token=token, ssl_verify=False)
         self.g.auth()
-        self.user = self.g.users.list(username=self.g.user.get_username())[0]
         self.repo = None
         if full_repo_name:
             self.repo = self.g.projects.get(full_repo_name)
 
+    @property
+    def user(self) -> GitUser:
+        return self.g.users.list(username=self.g.user.get_username())[0]
```

Comment:
Similarily to GitHub/Pagure, we want to have `GitlabUser` class -- this property should return an instance of that class.

We need to forward a service instance (`self`) when creating that instance. Same as we did for GitHub:

https://github.com/packit-service/ogr/blob/master/ogr/services/github.py#L140
```python
    @property
    def user(self) -> GitUser:
        return GithubUser(service=self)
```
https://github.com/packit-service/ogr/blob/master/ogr/services/github.py#L801
```python
class GithubUser(BaseGitUser):
    service: GithubService

    def __init__(self, service: GithubService) -> None:
        super().__init__(service=service)

# :
# :

    def get_username(self) -> str:
        return self.service.github.get_user().login
```